websites/M/Mugzone/presence.ts

```diff
@@ -0,0 +1,189 @@
+const presence = new Presence({
+  clientId: "823408394098311178"
+}),
+browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+const privacy = await presence.getSetting("privacy"),
+  presenceData: PresenceData = {
+    largeImageKey: "malody"
+  };
+if (privacy) {
+  presence.setTrayTitle();
+  presence.setActivity();
+} else {
```

Comment:
You could show text like browsing instead of nothing at all...

Reply:
I did that now!Thanks for the suggestion.